src/containerapp/azext_containerapp/custom.py

```diff
@@ -958,6 +963,7 @@ def update_containerappsjob(cmd,
                             remove_all_env_vars=False,
                             cpu=None,
                             memory=None,
+                            gpu=None,
```

Comment:
Why you update `create_containerappsjob` and `update_containerappsjob`? Do the job support gpu param?
If yes, please add test to cover commands `az containerapp job create/update --gpu`.